deeplearning4j-nn/src/main/java/org/deeplearning4j/optimize/solvers/accumulation/EncodedGradientsAccumulator.java

```diff
@@ -165,7 +165,7 @@ public void registerConsumers(int numConsumers) {
                 log.info("Master thread locks at RC");
 
             while (registered.get()) {
-                LockSupport.parkNanos(100L);
+                ThreadUtils.uncheckedSleep(1);
```

Comment:
Not sure it matters too much in practice, but we are changing the wait time by a few orders of magnitude here...

Reply:
I would argue this is better (I guess that's obvious since I provided the PR).

Few OS / schedulers will be able to park a thread for only 100 nanoseconds.  IME on linux you are looking at about 1000 nanoseconds in the least, and 1 millisecond at the most.  On windows 1ms tends to actually be optimistic.

I tend to target 1 millisecond because that will provide consistent behavior.  In addition by waking up more frequently you are furthering the context switching / cache miss problem that I described above.